commercetools-sdk-base/src/main/java/io/sphere/sdk/models/GeoJSON.java

```diff
@@ -0,0 +1,15 @@
+package io.sphere.sdk.models;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+/**
+ * Base class for GeoJSON objects that provides the required annotation
+ * to map to the correct sub type via the <code>type</code> property.
```

Comment:
Let me know if it's wrong, but shouldn't it be `{@code type}`, which behaves similarly in the generated Javadoc but belongs to the Javadoc specification? Sometimes we can't help it and we have to use HTML, but I would imagine this one is not the case.